src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/enable-serverless-monitoring-aws-lambda-layerless.mdx

```diff
@@ -169,8 +170,22 @@ If you're using a Lambda function without a layer, you'll need to instrument New
          callback();
        });
        ```
+       - Sample Code for ES Module:
```

Comment:
Suggestion: use "*" instead of "-" for bulleted list.
 example:  * Sample Code for ES module: